### PR TITLE
New package: python-twitter-2.2

### PR DESCRIPTION
**Pull request by pbui**

### CODE REVIEW

File: srcpkgs/python-twitter/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'python-twitter'
+pkgname=python-twitter
+version=2.2
+revision=1
+noarch=yes
+wrksrc="${pkgname}-${version}"
```
Comment (Vaelatern) [inline srcpkgs/python-twitter/template]: Line not necessary. `wrksrc` is this value by default.

File: srcpkgs/python-twitter/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+hostmakedepends="python-setuptools"
+depends="python python-requests python-requests-oauthlib"
+short_desc="A Python wrapper around the Twitter API (Python2)"
+maintainer="Peter Bui <pnutzh4x0r@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/bear/python-twitter"
+distfiles="${PYPI_SITE}/p/python-twitter/python-twitter-${version}.tar.gz"
+checksum=b70648c6026ef3772a1c911382d9fd8651fe0b673c15aad7d38d6f685c7c4dc2
+
+post_install() {
```
Comment (Vaelatern) [inline srcpkgs/python-twitter/template]: Function not needed. As I was told, `Apache-2.0` is one of those licenses that doesn't need `vlicense`.

**Comment by pbui** [comment]
@Vaelatern I've made the modifications.  Thanks for the feedback.

File: srcpkgs/python-twitter/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'python-twitter'
+pkgname=python-twitter
+version=2.2
+revision=1
+noarch=yes
+build_style=python-module
+python_versions="2.7"
```
Comment (xtraeme) [inline srcpkgs/python-twitter/template]: This line is redundant.

File: srcpkgs/python-twitter/template [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+pycompile_module="python-twitter"
```
Comment (xtraeme) [inline srcpkgs/python-twitter/template]: This is wrong, should be `twitter` module not `python-twitter`.

File: srcpkgs/python-twitter/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+hostmakedepends="python-setuptools"
+depends="python python-requests python-requests-oauthlib"
+short_desc="A Python wrapper around the Twitter API (Python2)"
+maintainer="Peter Bui <pnutzh4x0r@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/bear/python-twitter"
+distfiles="${PYPI_SITE}/p/python-twitter/python-twitter-${version}.tar.gz"
```
Comment (xtraeme) [inline srcpkgs/python-twitter/template]: use `${pkgname}`

**Comment by pbui** [comment]
@xtraeme I've made the modifications.  Thanks for the feedback.

File: srcpkgs/python-twitter/template [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'python-twitter'
+pkgname=python-twitter
+version=2.2
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="twitter"
+hostmakedepends="python-setuptools"
+depends="python python-requests python-requests-oauthlib"
+short_desc="A Python wrapper around the Twitter API (Python2)"
```
Comment (Vaelatern) [inline srcpkgs/python-twitter/template]: Please make: `Python2 wrapper around the Twitter API`

**Comment by asergi** [comment]
Merged manually in 0e9ee44.

**Pull request closed without merging**